### ISSUE TITLE
Request: Add unified documentation for slots / slot property

### ISSUE THREAD

**Issue by aprixon01112017** [quasarframework/quasar-framework.org#483]
I was looking for documentation about slots usable with tables.No luck.
I propose a dedicated paragraph about usable slots on each and every relevant component doc page.

**Comment by rossity** [comment]
Agreed. Apparently there are slots on the QField component (super useful to my use case) and they were not in docs, had to look through the component source.
